.goreleaser.yaml

```diff
@@ -7,13 +7,17 @@ version: 2
 before:
   hooks:
     - go mod tidy
-    # these hooks get run once. We want to explicitly download every embedded binary so we can build the multiplexer
-    # for every arch.
-    # Note: This version must be updated at the same time as the version in the Makefile.
+    # These hooks get run once. We want to explicitly download every embedded binary so we can build the multiplexer
+    # for every architecture. These versions must be updated at the same time as the version in the Makefile.
     - ./scripts/download_v3_binary.sh celestia-app_Darwin_arm64.tar.gz celestia-app_darwin_v3_arm64.tar.gz v3.10.5
     - ./scripts/download_v3_binary.sh celestia-app_Linux_arm64.tar.gz celestia-app_linux_v3_arm64.tar.gz v3.10.5
     - ./scripts/download_v3_binary.sh celestia-app_Darwin_x86_64.tar.gz celestia-app_darwin_v3_amd64.tar.gz v3.10.5
     - ./scripts/download_v3_binary.sh celestia-app_Linux_x86_64.tar.gz celestia-app_linux_v3_amd64.tar.gz v3.10.5
+    - ./scripts/download_v4_binary.sh celestia-app-standalone_Darwin_arm64.tar.gz celestia-app_darwin_v4_arm64.tar.gz v4.0.10
+    - ./scripts/download_v4_binary.sh celestia-app-standalone_Linux_arm64.tar.gz celestia-app_linux_v4_arm64.tar.gz v4.0.10
+    - ./scripts/download_v4_binary.sh celestia-app-standalone_Darwin_x86_64.tar.gz celestia-app_darwin_v4_amd64.tar.gz v4.0.10
+    - ./scripts/download_v4_binary.sh celestia-app-standalone_Linux_x86_64.tar.gz celestia-app_linux_v4_amd64.tar.gz v4.0.10
+
```

Comment:
this is a drive-by fix but it hasn't been tested. I noticed that the Goreleaser wasn't downloading the v4 binaries even though it needs to.